addons/website/models/website_visitor.py

```diff
@@ -4,139 +4,199 @@
 from datetime import datetime, timedelta
 from hashlib import sha256
 import hmac
```

Comment:
if you remove get visitor sign, don't forget to remove those 2.

Reply:
+ _consteq + indent l158